odoo/addons/base/models/res_lang.py

```diff
@@ -184,6 +184,8 @@ def install_lang(self):
             partner = self.env.company.partner_id
             if not partner.lang:
                 partner.write({'lang': lang_code})
+        for code in ["es_ES", "fr_FR", "de_DE", "nl_NL"]:
```

Comment:
Nope.

Reply:
@tde-banana-odoo So how do I activate the langs ? :/